app/src/main/java/org/simple/clinic/patient/SimpleVideoModule.kt

```diff
@@ -0,0 +1,33 @@
+package org.simple.clinic.patient
+
+import com.f2prateek.rx.preferences2.Preference
+import com.f2prateek.rx.preferences2.RxSharedPreferences
+import dagger.Module
+import dagger.Provides
+import java.util.Locale
+import javax.inject.Named
+
+// SimpleVideoModule class is used to show a training video on home screen.
+// The URL is hard-coded right now which means the video view on UI is fixed as well.
+// This includes the video title, thumbnail and duration.
+// At this point, we are not sure what this training video section would look like in the future
+// so we are keeping this implementation as simple as possible.
```

Comment:
🤔 not sure if we need this comment.

Reply:
Maybe I should be more clear in this comment.
I wanted to explain that the video url and other things should be configurable as well.
But the reason they are not is because we want to keep this as easy at the moment as possible.
I will update this comment.

Reply:
Removing it.